examples/gcoap/gcoap_cli.c

```diff
@@ -23,13 +23,34 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include "net/gcoap.h"
+#include "net/gcoap/gcoap.h"
 #include "od.h"
 #include "fmt.h"
 
 #define ENABLE_DEBUG 0
 #include "debug.h"
 
+#ifdef CONFIG_GCOAP_ENABLE_DTLS
+    #include "tinydtls_keys.h"
+    #include "net/credman.h"
+
+    #define SOCK_DTLS_CLIENT_TAG (2)
+    static const uint8_t psk_id_0[] = PSK_DEFAULT_IDENTITY;
+    static const uint8_t psk_key_0[] = PSK_DEFAULT_KEY;
+
+    static const credman_credential_t credential = {
+        .type = CREDMAN_TYPE_PSK,
+        .tag = SOCK_DTLS_CLIENT_TAG,
+        .params = {
+            .psk = {
+                .key = { .s = psk_key_0, .len = sizeof(psk_key_0) - 1, },
+                .id = { .s = psk_id_0, .len = sizeof(psk_id_0) - 1, },
+            }
+        },
+    };
```

Comment:
Please don't indent on pre-processor expressions. Not only is it not within our style (we don't indent guards either) AFAIK most compilers that don't like `#` prefixed with white-spaces.

Reply:
There are parts in RIOT which are barely readable because of the missing indents IMHO.  But when even some compilers complain about that I'm instantly convinced. Not necessary to spot more of those. I'll fix it!

Reply:
Maybe you don't even need those, when using `IS_ACTIVE()` ;-)